{{cookiecutter.repo_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# How to contribute
+
```

Comment:
As much as possible, i would love to have the contributing process centralized somewhere, instead of in copies of a file in each repo.  I think people will only find the centralized info in proportion to how short the CONTRIBUTING.md file is.  Can we move more of this info to the central document?